modules/custom/az_person/config/install/views.view.az_person.yml

```diff
@@ -305,15 +305,12 @@ display:
         - user.permissions
       tags:
         - 'config:field.storage.node.field_az_person_category'
-  grid:
-    display_plugin: page
-    id: grid
-    display_title: 'Grid Page'
-    position: 1
+  block_1:
+    id: block_1
```

Comment:
I recommend changing the machine name for this view display. Otherwise all looks good!